scripts/audioplot.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/python
+
+# audioplot is a simple script for drawing graphs of values.  It reads a simple
+# csv text file where each line needs to contain the same number of comma-separated values,
+# like so:
+# 0, 0, 0
+# 1, 2, 1
+# 2, 4, 2
+#
+# Each column will be plotted as a separate curve on the same time series.
+# Matplotlib is used to display the result.
+
+
+from matplotlib import pyplot as plt
+from matplotlib.lines import Line2D
+import time, string, glob, sys
+
+import getopt
```

Comment:
The standard Python way to parse cli options is argparse. Removes a lot of the boilerplate code. I'll write you a PR to make this script nicer